cilantro_audit/auditor_completed_audit_page.py

```diff
@@ -44,23 +44,24 @@ def reset_scroll_to_top(self):  # needs to be used in the routine that first pop
         self.scrolling_panel.scroll_y = 1
 
     def add_title(self, title):  # needs to be updated when you click out of one audit and load up another
-        lbl = Label(text='[b]Audit: [/b]' + title, markup=True, size_hint_y=None, height=40, halign="left")
+        lbl = Label(text='[b]Audit: [/b]' + title, markup=True, size_hint_y=None, height=40, font_size=20, halign="left")
         self.grid_list.add_widget(lbl)
 
     def add_auditor(self, auditor):  # needs to be updated when you click out of one audit and load up another
-        lbl = Label(text='[b]Auditor: [/b]' + auditor, markup=True, size_hint_y=None, height=40, halign="left")
+        lbl = Label(text='[b]Auditor: [/b]' + auditor, markup=True,
+                    size_hint_y=None, height=40, font_size=20, halign="left")
         self.grid_list.add_widget(lbl)
 
     def add_date_time(self, dt):  # needs to be updated when you click out of one audit and load up another
-        lbl = Label(text='[b]Date: [/b]' + dt, markup=True, size_hint_y=None, height=40, halign="left")
+        lbl = Label(text='[b]Date: [/b]' + dt, markup=True, size_hint_y=None, height=40, font_size=20, halign="left")
         self.grid_list.add_widget(lbl)
 
     def add_blank_label(self, text):
-        lbl = Label(text=text, size_hint_y=None, height=40, halign="left")
+        lbl = Label(text=text, size_hint_y=None, height=40, font_size=20, halign="left")
         self.grid_list.add_widget(lbl)
 
-    def add_question_answer_auditor_version(self, answer):
-        self.stack_list.height += 80  # integer (80) comes from question_answer size
+    def add_question_answer_auditor_version(self, question, answer):
```

Comment:
`question` parameter not used in this method, and its addition now crashes when loading a completed audit from the auditor menu.

Reply:
Resolved